### DISCUSSION TITLE
[@types/jquery] return type of method "children()"

### DISCUSSION THREAD

**Discussion by danielrentz** [DefinitelyTyped/DefinitelyTyped#57409] [Issues with a @types package]
Currently, `JQuery#children()` defines "this" as its return type, but this is not true, as the element type may most likely change.

For example, the return type of a `JQuery<HTMLSelectElement>` is likely a `JQuery<HTMLOptionElement>`, not `this`.

The signature should be adjusted according to the method `JQuery#find()`.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
Hi @danielrentz. You are right. There is a lot of similar issues. I will create PR.

**Reply by danielrentz** [reply]
thanks!

**Reply by martin-badin** [reply]
#57421
